sklearn_numba_dpex/common/random.py

```diff
@@ -0,0 +1,232 @@
+import warnings
+import random
+import dpctl
+from functools import lru_cache
+from numba import float32, float64, uint32, int64, uint64
+
+import numba_dpex as dpex
+
+import numpy as np
+
+# This code is largely inspired from the numba.cuda.random module and the
+# numba/cuda/random.py where it's defined (v<0.57).
```

Comment:
Thank you for indicating this. Can you include a permalink to those implementations?